### ISSUE TITLE
[Bug] Switch ThumbColor is broken

### ISSUE THREAD

**Issue by mfeingol** [xamarin/Xamarin.Forms#8258]
### Description

There are two problems, at least on Android:

1. When a switch is initialized toggled, the 'on' thumb color is always the accent color. But when a switch is initialized untoggled, the 'on' thumb color is some shade of white. So two otherwise identical switches will look different for no obvious reason.
2. A switch's thumb actually has two colors, an 'on' color and an 'off' color. However, Xamarin.Forms.Switch only offers a single property, ThumbColor, which overrides both the on and off states. Ideally, this would be broken up into two distinct properties.

### Steps to Reproduce

```
        <Switch IsToggled="True" />
        <Switch />
        <Switch ThumbColor="Blue" />
```

### Expected Behavior

1. Switch thumb colors shouldn't depend on the initial toggled state of the switch.
2. We should be able to independently control both the on and off colors for the thumb.

### Actual Behavior

As described.

### Basic Information

- Version with issue: 4.3.0.908675
- Last known good version: Unknown
- IDE: VS 2019 16.3.6
- Platform Target Frameworks: 
  - Android: 9.0
- Affected Devices: Android

### Screenshots

![image](https://user-images.githubusercontent.com/218865/67629664-47522900-f836-11e9-8061-f1e452ebb904.png)

### Reproduction Link

https://github.com/mfeingol/repros/tree/master/SwitchOnColor-8258

**Comment by techduggu** [comment]
Possibly duplicate of #7249 and is fixed with #7777

**Comment by hartez** [comment]
@techduggu is correct - problem 1 is the same as #7249.

For problem 2 - instead of adding a separate property for the thumb color when the control is in the "On" state, we should allow that to be managed by the VSM. I've created an issue for that: https://github.com/xamarin/Xamarin.Forms/issues/8263
